tests/10-create.js

```diff
@@ -0,0 +1,286 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+
+import {generateMultibase, splitDid} from './helpers.js';
+import {
+  shouldBeDidResolverResponse,
+  shouldBeValidDid,
+  shouldErrorWithData,
+  shouldHaveDidDereferencingError,
+  shouldHaveDidResolutionError,
+  shouldHaveValidVersion
+} from './assertions.js';
+import chai from 'chai';
+import {filterByTag} from 'vc-api-test-suite-implementations';
+
+const should = chai.should();
+const headers = {
+  Accept: 'application/ld+json;profile="https://w3id.org/did-resolution"'
+};
```

Comment:
Just thinking out loud: don't know if I like this yet... but understand it's currently in the DID Resolution spec.

Reply:
yeah the did resolver spec in general is in a bad state I'd say. So many todos. Maybe we should make an issue and discuss this with @peacekeeper in a Monday meeting?